README.md

```diff
@@ -1,6 +1,6 @@
 # Swagger Gradle Codegen
 
-[![Plugin Portal](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/com/yelp/codegen/plugin/com.yelp.codegen.plugin.gradle.plugin/maven-metadata.xml.svg?label=Gradle%20Plugin%20Portal&colorB=brightgreen)](https://plugins.gradle.org/plugin/com.yelp.codegen.plugin) [![Build Status](https://travis-ci.com/Yelp/swagger-gradle-codegen.svg?branch=master)](https://travis-ci.com/Yelp/swagger-gradle-codegen) [![License](https://img.shields.io/badge/license-Apache2.0%20License-orange.svg)](https://opensource.org/licenses/Apache2.0) [![Twitter](https://img.shields.io/badge/Twitter-@YelpEngineering-blue.svg)](https://twitter.com/YelpEngineering)
+[![Plugin Portal](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/com/yelp/codegen/plugin/com.yelp.codegen.plugin.gradle.plugin/maven-metadata.xml.svg?label=Gradle%20Plugin%20Portal&colorB=brightgreen)](https://plugins.gradle.org/plugin/com.yelp.codegen.plugin) [![Join the chat at https://kotlinlang.slack.com](https://img.shields.io/badge/slack-@kotlinlang/swagger--gradle--codegen-yellow.svg?logo=slack)](https://kotlinlang.slack.com/archives/CU233PG2Z) [![Build Status](https://travis-ci.com/Yelp/swagger-gradle-codegen.svg?branch=master)](https://travis-ci.com/Yelp/swagger-gradle-codegen) [![License](https://img.shields.io/badge/license-Apache2.0%20License-orange.svg)](https://opensource.org/licenses/Apache2.0) [![Twitter](https://img.shields.io/badge/Twitter-@YelpEngineering-blue.svg)](https://twitter.com/YelpEngineering)
```

Comment:
nit. I would put closer to the Twitter account badge
As user of open-source libraries the first thing that I check is "does it have green tests? what are the licences? where can I find it? and then eventually communication mediums"